duo_client/admin.py

```diff
@@ -2523,14 +2523,13 @@ def get_admin(self, admin_id):
         response = self.json_api_call('GET', path, {})
         return response
 
-    def add_admin(self, name, email, phone, password, role=None):
```

Comment:
These arguments are deprecated and are ignored by the api

Reply:
I'm trying to think through the ramifications of making this change. If it's deprecated upstream we definitely need to fix the client to not provide it. But I'm thinking this means we need to do some kind of version bump to clarify that people's clients might start breaking if they are passing in that argument.
@vbscott @AaronAtDuo thoughts?

Reply:
Maybe we should start by seeing if people are providing that field and raise a Deprecation warning of some kind?

Reply:
There are still people providing that field.  I like the Deprecation warning, is there any existing pattern of doing this?

Reply:
So we definitely can't just remove it unless we do a major version bump, and I don't vote that we do that now.  FWIW the parameter is already documented as deprecated in the Admin API docs.  So I think we:
- Leave the parameter in the method signature (i.e. revert the change in line 2526)
- Indicate it's deprecated in the docstring (line 2533)
- Ignore it in the code (i.e. keep the change to lines 2547+2548)

Reply:
Okay!